pkg/actuators/machine/awsclient.go

```diff
@@ -128,3 +128,28 @@ func (client *AwsClientWrapper) GetAvailabilityZone(machine *machinev1beta1.Mach
 	}
 	return *instance.Placement.AvailabilityZone, nil
 }
+
+// GetVolumes gets volumes attached to instance
+func (client *AwsClientWrapper) GetVolumes(machine *machinev1beta1.Machine) (map[string]map[string]interface{}, error) {
```

Comment:
I won't block this PR on this, but I really think we should at least make a note to return a real type here. Whether that is just the block device mapping type from the AWS SDK, or we make a simple struct of our own.

Reply:
That makes sense, will take a note as follow up.